zetaclient/zetabridge/tx.go

```diff
@@ -151,19 +151,19 @@ func (b *ZetaCoreBridge) SetTSS(tssPubkey string, keyGenZetaHeight int64, status
 	return "", fmt.Errorf("set tss failed | err %s", err.Error())
 }
 
-func (b *ZetaCoreBridge) ConfigUpdater(cfg *config.Config) {
-	b.logger.Info().Msg("ConfigUpdater started")
-	ticker := time.NewTicker(time.Duration(cfg.ConfigUpdateTicker) * time.Second)
+func (b *ZetaCoreBridge) CoreContextUpdater(appContext *appcontext.AppContext) {
```

Comment:
Although it wasn't added so far, I think it's a good opportunity to add comment for the method